### DISCUSSION TITLE
Combine a static source with a Muellmax source

### DISCUSSION THREAD

**Discussion by froggi1930** [mampfes/hacs_waste_collection_schedule#1764] [Q&A]
I am trying to combine those two sources, but it fails. Is it possible to mix those, or do I need to change anything?

```
# waste_collection_schedule source configuration
waste_collection_schedule:
  sources:
    - name: muellmax_de
      args:
        service: Tbr
        mm_frm_ort_sel: xxx
        mm_frm_str_sel: xxx
        mm_frm_hnr_sel: xxx;xxx;xxx;xxx

    - name: static
      args:
        type: Gelber Sack
        frequency: WEEKLY
        inerval: 2
        start: "2024_02-09"
        until: "2024-12-31"

**Comment by 5ila5** [comment]
This should work. You will get two calendar entities. When using sensors, you need to pass the source_index argument  (`source_index: 0` for muellmax source (optional as this is the default value), `source_index: 1` for static source or `source_index: [0,1]` for a sensor accessing both sources)

What exactly do you mean with "it fails"? If you still have any problems, please provide logs or a more detailed explanation of your error.

**Reply by froggi1930** [reply]
Sorry - I could not reply before...
If I use Muellmax as stand alone I can see the calender, but if I use the static alone or in combination with Muellmax I do not see any calender at all.
Where can I find the log files, so that I can search for an error message?

**Reply by 5ila5** [reply]
settings -> system -> logs
